README.rdoc

```diff
@@ -2,12 +2,16 @@
 
 1. Clone it
 
-2. rake db:migrate
+2. Rename database.yml.example to database.yml (or customize)
 
-3. rake db:seed
+3. Rename secrets.yml.example to secrets.yml (or generate a new one)
 
-4. rails s
+4. rake db:setup
```

Comment:
I like rake db:setup, since you get create (which was missing from the readme), migrate, and seed all in one command. You also get schema load, but that doesn't do anything in this case.